evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -281,6 +281,57 @@ contract Operator is
     s_forwarder.setAuthorizedSenders(forwarderSenders);
   }
 
+  /**
+   * @notice If the s_forwarder is owned by a different address, deploy and set a new one
+   */
+  function deployForwarder()
+    external
+    override
+    onlyOwner()
+    returns (
+      address
+    )
+  {
+    require(address(this) != s_forwarder.owner(), "Operator is forwarder owner");
+    OperatorForwarder newForwarder = new OperatorForwarder(address(linkToken));
+    s_forwarder = newForwarder;
```

Comment:
Do you want to get rid of the single forwarder slot in this PR or the next one?

Reply:
Let's do the next one.